code/modules/mob/living/simple_animal/hostile/headslug.dm

```diff
@@ -1,17 +1,18 @@
-#define EGG_INCUBATION_DEAD_TIME 120
-#define EGG_INCUBATION_LIVING_TIME 200
+#define EGG_INCUBATION_DEAD_TIME 60
+#define EGG_INCUBATION_LIVING_TIME 120
```

Comment:
Do you think you could rename some of the variables here to be `cycles` vs `time`? It's a little confusing that this is in reference to ticks of SSObj while the variable keeping track of things is called `time`.

Reply:
Change pushed lewc.